shanoir-ng-datasets/src/main/java/org/shanoir/ng/download/WADODownloaderService.java

```diff
@@ -399,26 +351,25 @@ public AcquisitionAttributes<Long> getDicomAttributesForAcquisition(DatasetAcqui
 		return dAcquisitionAttributes;
 	}
 
-	static String extractInstanceUID(String url) {
```

Comment:
@jcomedouteau I think StringUtils have better performances than regexp here. I think I even measured it.

Do you think the code was malfunctioning ?

Reply:
No, I think it's a mistake from the merge (or from my PR I'm not sure), but we need to keep your code !